### PR TITLE
Improving the tiled sprite object page.

### PR DESCRIPTION
**Pull request by HelperWesley**
-Updating the images
-Updating language

### CODE REVIEW

**Comment by HelperWesley** [comment]
Final and should be ready for review.

File: docs/gdevelop5/objects/tiled_sprite.md [hunk 1]
```diff
@@ -9,67 +9,57 @@ title: Tiled Sprite
...
 ## Examples
 
 !!! tip
 
         **See it in action!** 🎮
-    Open these examples online.
+    Open this example online.
 
-[![](/gdevelop5/behaviors/platformerbehavior.png)](https://editor.gdevelop.io/?project=example://old-platformer)
+[![](/gdevelop5/behaviors/Examples1PlatformerTiledSprite.png)](https://editor.gdevelop.io/?project=example://old-platformer)
```
Comment (AlexandreSi) [inline docs/gdevelop5/objects/tiled_sprite.md:63]: Here, the url should be `/gdevelop5/objects/Examples1PlatformerTiledSprite.png`

Comment (AlexandreSi) [inline docs/gdevelop5/objects/tiled_sprite.md:63]:
```suggestion
[![](/gdevelop5/objects/Examples1PlatformerTiledSprite.png)](https://editor.gdevelop.io/?project=example://old-platformer)
```

**Review by AlexandreSi — APPROVED** [review]
LGTM

**Pull request merged**